### DISCUSSION TITLE
Chords working, but not Shorthand

### DISCUSSION THREAD

**Discussion by Hardy02** [psoukie/zipchord#209] [Q&A] [answered]
I can do the chording easily, and it works well; however, when I try to do the shorthand feature, it doesn't work.
I have tried disabling chords and only having shorthand ticked, but this did not fix the issue. 
Any suggestions as to why?

**Comment by Hardy02** [comment, marked as answer]
All good! It was because I had "Expand shorthand immediately" selected.

**Reply by jsampson45** [reply]
Not good here though. Selecting or deselecting "Expand shorthand immediately" makes no difference.

**Reply by psoukie** [reply]
Could you provide more details on what does not seem to make a difference? When I test it and type `F` `L` `Space` I get `feel` (when Expand shorthands immediately is disabled). When it's enabled, it expands right after F and L. (Which is why I don't have it as default because for many dictionaries, it produces unintended results. In this example, a shorthand such as `fl` prevents typing things like "fly".)

**Comment by jsampson45** [comment]
It is difficult. If I press "edit", Notepad comes up and in it shorthands expand. I have found that they expand in Microsoft Word. There may be other environments where they expand but these do not include Chrome or other text editors I have tried.
Kind regards
John S.
On 18/07/2024 02:16:14, Pavel Soukenik ***@***.***> wrote:
Could you provide more details on what does not seem to make a difference? When I test it and type F L Space I get feel (when Expand shorthands immediately is disabled). When it's enabled, it expands right after F and L. (Which is why I don't have it as default because for many dictionaries, it produces unintended results. In this example, a shorthand such as fl prevents typing things like "fly".)
—
Reply to this email directly, view it on GitHub [https://github.com/psoukie/zipchord/discussions/209#discussioncomment-10078757], or unsubscribe [https://github.com/notifications/unsubscribe-auth/APUQLDWL6ILKT5MPQM2H5SLZM4JNBAVCNFSM6AAAAABJXDKFU2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMBXHA3TKNY].
You are receiving this because you commented.Message ID: ***@***.***>
[43747703-b2d2-4210-ae5c-4d79aeac6023]
